qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,134 @@
+Feature: 1.1. Search by Filters
+
+Background: 
+    Given User is on https://career.quantori.com website
+    And There are 10 available <positions> on the jobs page
+    And There are 2 available <competence> on the jobs page
+    And There are 2 available <technologies> on the jobs page
+    |Position                                              ||competence       ||technologies|
+    |Junior JavaScript Engineer                            ||Databases        ||SQL         |
+    |Senior/Lead Manual QA Engineer (Life Science projects)||Business Analysis||.NET        |
+    |Senior/Lead Manual QA Engineer (internal projects)    ||                 ||            |
+    |Data Engineer (SQL)                                   ||                 ||            |
+    |Snowflake Data Warehouse Certified Engineer           ||                 ||            |
+    |Senior DWH / ETL Data Engineer                        ||                 ||            |
+    |Data Engineer (Spark)                                 ||                 ||            |
+    |Junior JavaScript Engineer                            ||                 ||            |
+    |Senior QA Automation (Python) Engineer / SDET         ||                 ||            |
+    |Senior Data Engineer                                  ||                 ||            |
+
+
+Scenario: 1.1_01 Checking the function of the search button
+    When User pushes "search" button
+    Then User should be redirected to the new page "https://career.quantori.com/ru/positions"
```

Comment:
Then user should be redirected to "https://career.quantori.com/ru/positions" page 
OR
Then "https://career.quantori.com/ru/positions" page should be opened (it's shorter)